descriptors/ModuleDescriptor-template.json

```diff
@@ -168,60 +168,6 @@
           "pathPattern" : "/_/tenant"
         }
       ]
-    },
-    {
-      "id": "patron-block-conditions",
-      "version": "0.1",
-      "handlers": [
-        {
-          "methods": ["GET"],
```

Comment:
Should we update module version when we remove interfaces from it?

Reply:
Updated version of module to 16.2.1

Reply:
Updated version of module to 16.2.1